Dockerfile

```diff
@@ -153,7 +153,7 @@ ARG CARGO_BUILD_JOBS=default
 ENV CARGO_BUILD_JOBS=$CARGO_BUILD_JOBS
 
 # Install AVM - Anchor version manager for Solana
-RUN cargo install --git https://github.com/coral-xyz/anchor avm
+RUN cargo install --git https://github.com/coral-xyz/anchor avm --tag v0.30.1
```

Comment:
I think the version tag should be made 0.29.0 to align with the needed Anchor version